### PR TITLE
Add more up-to-date datetimepicker

### PR DESCRIPTION
**Pull request by cameronblandford**
Also eliminates a bower dependency that was causing a lot of bugs.

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -34,11 +34,8 @@ module.exports = function(defaults) {
     app.import('bower_components/loaders.css/loaders.min.css');
     app.import('bower_components/hint.css/hint.min.css');
     app.import('bower_components/jquery.tagsinput/src/jquery.tagsinput.js');
-    app.import('bower_components/moment/min/moment.min.js');
     app.import('vendor/assets/ember-osf.css');
-
-
-    app.import()
+    app.import();
```
Comment (birdbrained) [inline ember-cli-build.js]: Is this doing anything?

File: package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-jsoneditor": "0.1.4",
-    "ember-cli-moment-shim": "3.3.3",
+    "ember-cli-moment-shim": "3.3.0",
```
Comment (birdbrained) [inline package.json]: We had to downgrade?

File: package.json [hunk 3]
```diff
@@ -52,7 +52,7 @@
     "ember-i18n": "^5.0.2",
     "ember-load-initializers": "^0.5.1",
     "ember-lodash": "^4.17.5",
-    "ember-moment": "7.3.1",
+    "ember-moment": "^7.4.1",
```
Comment (birdbrained) [inline package.json]: 👍

File: package.json [hunk 4]
```diff
@@ -68,11 +68,14 @@
...
     "lodash.omit": "^4.5.0",
-    "moment": "^2.18.1",
     "pagination-pager": "2.4.2",
     "regenerator": "^0.8.46"
   },
   "devDependencies": {
-    "ember-simple-auth": "^1.4.0"
+    "ember-cli-bootstrap-datetimepicker": "^0.5.0",
+    "ember-simple-auth": "^1.4.0",
+    "eonasdan-bootstrap-datetimepicker-npm": "4.17.42",
```
Comment (birdbrained) [inline package.json]: Is this deprecated?

Comment (cameronblandford) [reply package.json]: my guess is that ember-cli-bootstrap-datetimepicker might be a wrapper around the eonasdan one?

File: ember-cli-build.js (hunk 1 above)
Comment (cameronblandford) [reply ember-cli-build.js]: Keeps tests from failing!

File: package.json (hunk 2 above)
Comment (cameronblandford) [reply package.json]: 3.3.0 is the version that ember-cli-bootstrap-datetimepicker uses. I can try it with 3.3.3?

**Pull request merged**